File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterResult.scala

package org.openapitools.client.model


case class ResponseTimeFilterResult (
    _searchId: String,
    _locations: List[ResponseTimeFilterLocation],
    _unreachable: List[String]
)
object ResponseTimeFilterResult {
    def toStringBody(var_searchId: Object, var_locations: Object, var_unreachable: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"locations":$var_locations,"unreachable":$var_unreachable
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/OpenAPI/src/model/ResponseTimeFilterResult.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.ResponseTimeFilterLocation

module ResponseTimeFilterResult = 

  //#region ResponseTimeFilterResult


  type ResponseTimeFilterResult = {
    SearchId : string;
    Locations : ResponseTimeFilterLocation[];
    Unreachable : string[];
  }
  //#endregion
  